src/components/pages/gallery/Upload.tsx

```diff
@@ -84,32 +100,51 @@ export default function Upload(props: Props) {
             },
             props.setFiles
         );
+
+        if (isElectron()) {
+            ImportService.getPendingUploads().then(
+                ({ files, collectionName }) => {
```

Comment:
super nit: `files` -> `electronFiles`. The term file is already overloaded with system file, enteFile and now electronFile :)